control/content/app.services.js

```diff
@@ -362,7 +362,7 @@
               userMessage: `Generate a contact us information related to [business-type] located in [target-region].\nFor phone number use [+1 555 555-1234].\nFor email use [${user?.email || ''}].`,
               maxRecords: 5,
               systemMessage:
-                  'images are two 1080x720 images URLs related to location, use source.unsplash.com for images, URL should not have premium_photo or source.unsplash.com/random. return description as HTML',
+                  'images are two  images URLs related to location, use https://app.buildfire.com/api/stockImages/?topic=&imageType=medium , use business-type or target region as a topic for images. return description as HTML',
```

Comment:
imagesURLs are two images related to the business type or the target region, topic should be transformed to kebab-case, use https://app.buildfire.com/api/stockImages/?topic={topic}&imageType=medium. return description as HTML',